packages/loader/driver-definitions/src/storage.ts

```diff
@@ -59,8 +69,16 @@ export interface IDeltaStorageService {
 export interface IDocumentDeltaStorageService {
     /**
      * Retrieves all the delta operations within the exclusive sequence number range
-     */
-    get(from: number, to: number): Promise<IDeltasFetchResult>;
+     * @param from - first op to retrieve (inclusive)
+     * @param to - first op not to retrieve (exclusive end)
+     * @param abortSignal - signal that aborts operation
+     * @param cachedOnly - return only cached ops, i.e. ops available locally on client.
+     */
+     fetchMessages(from: number,
+        to: number | undefined,
+        abortSignal?: AbortSignal,
+        cachedOnly?: boolean,
+    ): IStream<ISequencedDocumentMessage[]>;
```

Comment:
Should be make this an "async iterator" since they are so similar already?

Reply:
I thought about it. Maybe. I have a feeling that it will be useful to be able to query it for more things. Like, have progress reporting (i.e. how many ops are still there, if that can be answered). Maybe we can have it both?
I do not have strong opinion here, happy to convert to async iterator.

The part that worries me is red X next to Safari on iOS:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/asyncIterator
Can't find equivalent, but I had to back out code earlier due to usage of async generators due to compat concerns, so I'm a bit hesitant to jump here.

Reply:
I've done manual test on iOS and async iterator is working fine for me, being on latest iOS.
That said, https://caniuse.com/?search=async%20iterator also is in red. So I'm not sure.
I'm thinking of leaving it as is for now (though ask around), until it becomes green across the board (yes, future changes would require back compat, but it's not such a big deal imho). Let me know what you think